### DISCUSSION TITLE
Changing the Quarkus loggers level from Unleash

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#39864] [Quarkus Blog/Website]
# Changing the Quarkus loggers level from Unleash

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/changing-loggers-level-from-unleash/

**Comment by bulldog98** [comment]
Interesting the described workflow is also usable for other log level changing mechanisms.

**Comment by AnakinPt** [comment]
After 4 or 5 years, Quarkus still needs to have hacks to change the log level of a running container? No one knows something like Spring Boot Admin Console to manage a running application?

**Reply by gwenneg** [reply]
As mentioned in the opening section of this post, Quarkus provides a method to adjust logger levels via a REST API: https://github.com/quarkiverse/quarkus-logging-manager. However, this approach requires exposing a REST API which is not always an option.

**Reply by AnakinPt** [reply]
Does this change the log levels to all pods of the same deployment, or only for the pod that received the request? An Yes answer to this makes Quarkus to be Garbage in terms of operational usage.
